content/millennium/r4/financial/financial-transaction.md

```diff
@@ -0,0 +1,214 @@
+ `financial-transaction-allocation`      | None (contains nested extensions)      | Defines how the payment or adjustment is to be allocated across other resources.
+ `financial-transaction-amount`          | [`Money`]           | The total amount of the financial transaction.
+ `financial-transaction-card-brand`      | [`string`]          | Identifies the brand of credit card when credit card is used as a payment method.
+ `financial-transaction-date`            | [`date`]            | Represents the expiration date if method is card, check date if method is check, EFT date if method is EFT or Lockbox date if method is lockbox.
+ `financial-transaction-location`        | [`string`]          | Client configured value representing the location or workflow that the payment was received in.
+ `financial-transaction-method`          | [`string`]          | Describes the method of payment for the financial transaction.
+ `financial-transaction-tendered-amount` | [`Money`]           | The amount of cash originally tendered for payment. This value should be greater than or equal to the amount of the cash payment.
+ `financial-transaction-type`            | [`CodeableConcept`] | The classification of the transaction.
+
+## Create
+
+Create a FinancialTransaction.
+
+    POST /Basic    
+
+_Implementation Notes_
+
+* Only the body fields mentioned below are supported.
+* When integrating your application with a client's production environment, the client will have to provide the appropriate financial-transaction-alias values to send with the financial transaction.
+* Clients must use Cerner's Revenue Cycle Patient Accounting application in order to create Financial Transactions.
+* A FinancialTransaction can only be targeted at a clinical Encounter that has related financial activity within the Cerner Revenue Cycle Patient Accounting system.
+* FinancialTransaction and Basic scopes are required.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token&gt;', 'Content-Type': 'application/fhir+json'} %>
+
+### Body Fields
+
+<%= definition_table(:financial_transaction, :create, :r4) %>
+
+### Example - Cash Payment
+
+#### Request
+
+    POST /Basic
+
+#### Body
+
+  <%= json(:R4_FINANCIAL_TRANSACTION_CASH) %>
+
+#### Response
+
+<%= headers status: 201 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: application/fhir+json
+Date: Mon, 04 Nov 2019 16:23:57 GMT
+Etag: W/"0"
+Last-Modified: Mon, 04 Nov 2019 16:23:57 GMT
+Location: https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Basic/FT-PL-14266754
+Server-Response-Time: 3890.363996
+Status: 201 Created
+Vary: Origin
+X-Request-Id: 1638e30e497b93ff4383b2ff0eaeea68
+X-Runtime: 3.890282
+</pre>
+
+The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
```

Comment:
lets get rid of this line for the examples, and add the disclaimer: https://github.com/cerner/fhir.cerner.com/blame/master/content/millennium/r4/general-clinical/condition.md#L77

This should add a note saying the examples may not always return the same result.

Reply:
Added disclaimer
https://github.com/cerner/fhir.cerner.com/pull/297/commits/8f6113a4a9f782067bacf34ffdbe746768ad7cee#diff-a2a1182a3c7ddb3de2fa0458e90ddf0cR102